TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayerCore.java

```diff
@@ -185,9 +187,9 @@ Void handleCompletedTransaction(@NonNull IReplayContexts.IReplayerHttpTransactio
             } finally {
                 var requestKey = context.getReplayerRequestKey();
                 requestWorkTracker.remove(requestKey);
-                log.trace("removed rrPair.requestData to " +
-                        "targetTransactionInProgressMap for " +
-                        requestKey);
+                log.atTrace().setMessage("removed rrPair.requestData to " +
```

Comment:
Strange line break

Reply:
I've since simplified this change to be only a few changes that are bugs until discussion is finalized for #633